core/config/dev.exs

```diff
@@ -33,14 +33,9 @@ config :core, CoreWeb.Endpoint,
     ]
   ]
 
+config :core, Core.ImageCatalog.Unsplash,
+  access_key: System.get_env("UNSPLASH_ACCESS_KEY"),
+  app_name: System.get_env("UNSPLASH_APP_NAME")
+
 config :exsync,
   addition_dirs: ["../../frameworks"]
-
-# Load developer machine specific config. This can be used to setup secrets and
-# such to connect with 3rd party services.
-try do
-  import_config "dev.local.exs"
```

Comment:
The idea of the `dev.local.exs` file was so that configuration could be added with secret for development purposes. Env vars could also work for this; but then we need to set them in a convenient way. What do you prefer?

Reply:
I removed this code to prevent te warning since I had no dev.local.exs anymore. But I have no objection to use it. If needed we can introduce it again.